rmgpy/statmech/torsionTest.py

```diff
@@ -397,8 +406,9 @@ def test_getSumOfStates_quantum_fourier(self):
         sumStates = self.mode.getSumOfStates(Elist)
         densStates = self.mode.getDensityOfStates(Elist)
         for n in range(10, len(Elist)):
-            self.assertTrue(0.8 < numpy.sum(densStates[0:n]) / sumStates[n-1] < 1.25, '{0} != {1}'.format(numpy.sum(densStates[0:n]), sumStates[n]))
-        
+            self.assertTrue(0.8 < numpy.sum(
+                densStates[0:n]) / sumStates[n-1] < 1.25, '{0} != {1}'.format(numpy.sum(densStates[0:n]), sumStates[n]))
```

Comment:
This should be split after the comma rather than in the middle of the expression.

Reply:
Changed.